ui/src/views/Settings.vue

```diff
@@ -3,45 +3,110 @@
   SPDX-License-Identifier: GPL-3.0-or-later
 -->
 <template>
-  <cv-grid fullWidth>
-    <cv-row>
-      <cv-column class="page-title">
+  <div class="bx--grid bx--grid--full-width">
+    <div class="bx--row">
+      <div class="bx--col-lg-16 page-title">
         <h2>{{ $t("settings.title") }}</h2>
-      </cv-column>
-    </cv-row>
-    <cv-row v-if="error.getConfiguration">
-      <cv-column>
+      </div>
+    </div>
+    <div v-if="error.getConfiguration" class="bx--row">
+      <div class="bx--col">
         <NsInlineNotification
           kind="error"
           :title="$t('action.get-configuration')"
           :description="error.getConfiguration"
           :showCloseButton="false"
         />
-      </cv-column>
-    </cv-row>
-    <cv-row>
-      <cv-column>
-        <cv-tile light>
+      </div>
+    </div>
+    <div class="bx--row">
+      <div class="bx--col-lg-16">
+        <cv-tile :light="true">
           <cv-form @submit.prevent="configureModule">
-            <!-- TODO remove test field and code configuration fields -->
-            <cv-text-input
-              :label="$t('settings.test_filed')"
-              v-model="testField"
-              :placeholder="$t('settings.test_filed')"
+            <NsTextInput
+              :label="$t('settings.piler_fqdn')"
+              :placeholder="$t('settings.placeholder_piler_domain')"
+              v-model.trim="host"
+              :invalid-message="$t(error.host)"
               :disabled="loading.getConfiguration || loading.configureModule"
-              :invalid-message="error.testField"
-              ref="testField"
-            ></cv-text-input>
-            <cv-row v-if="error.configureModule">
-              <cv-column>
+              ref="host"
+            >
+            </NsTextInput>
+            <template v-if="host">
+            <span class="mg-bottom"> 
```

Comment:
Let's show **SMTP TCP port** inside a disabled `NsTextInput` (it supports tooltip), so we can have a more consistent style